dspace-api/src/main/java/org/dspace/core/Utils.java

```diff
@@ -506,4 +509,48 @@ public static String interpolateConfigsInString(String string) {
         ConfigurationService config = DSpaceServicesFactory.getInstance().getConfigurationService();
         return StringSubstitutor.replace(string, config.getProperties());
     }
+
+    /**
+     * Get the maximum timestamp that can be stored in a PostgreSQL database with hibernate,
+     * for our "distant future" access expiry date.
+     * @return the maximum timestamp that can be stored with Postgres + Hibernate
+     */
+    public static Instant getMaxTimestamp() {
```

Comment:
@max-nuding : Just a quick note... this other backport PR also seems to include the same unused methods?

(I haven't tested this 7.x backport yet, but I'm hoping to do so tomorrow or Thurs)